3.random_forest/README.md

```diff
@@ -0,0 +1,35 @@
+# random forest
+
+Random forest, collection of random generated desicion trees, is a form of classifier. The classification always goes via a hard cut in a feature. This work well if all features are independed.
```

Comment:
```suggestion
A random forest, a collection of random generated decision trees, is a form of a classifier.
The classification always goes via a hard cut in a feature.
This work well if all features are independent.
```

I'm firmly in the one-sentence-per-line markdown camp - https://github.com/prettier/prettier/issues/3302

It makes markdown easier to review on github and doesn't change formatting

Reply:
I can agree with that camp